# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gtksourceview-sharp/dont_install_vbnet.lang.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Our gtksourceview already have a vbnet.lang and nemerle.lang
Alejandro

--- ./Makefile.am.orig	2005-03-12 06:56:49.000000000 -0300
+++ ./Makefile.am	2005-03-24 17:30:46.985476048 -0400
@@ -1,9 +1,9 @@
 
 SUBDIRS=		gtksourceview sample doc 
-EXTRA_DIST=		nemerle.lang vbnet.lang gtksourceview-sharp-2.0.pc.in gtk-sharp.snk
+EXTRA_DIST=		gtksourceview-sharp-2.0.pc.in gtk-sharp.snk
 
 extra_langdir= $(gtksourceview_prefix)/share/gtksourceview-1.0/language-specs
-extra_lang_DATA=	nemerle.lang vbnet.lang
+extra_lang_DATA=	
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = gtksourceview-sharp-2.0.pc
